### PR TITLE
Content - added basic html structure

### PR DESCRIPTION
**Pull request by Knm24**
added basic html structure

### CODE REVIEW

**Comment by Knm24** [comment]
added basic html structure, please after review add

**Review by Annurkenova — APPROVED** [review]
I agree

File: index.html [hunk 1]
```diff
@@ -2,12 +2,30 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```
[the remark this answers is not in the document]
Comment (Knm24) [reply index.html:5]: yes

File: index.html [hunk 2]
```diff
@@ -2,12 +2,36 @@
...
+
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + TS</title>
+    <title>team3-skilled-elearning-landing-page</title>
+
+    <style>
+      @import url("/assets/styles/default.css");
+      @import url('/assets/styles/header.css');
+      @import url('/assets/styles/content.css');
+      @import url('/assets/styles/footer.css');
```
Comment (talgat-ruby) [inline index.html:13]: это можно делать в `main.ts`

**Review by talgat-ruby — APPROVED** [review]
LGTM!

**Comment by talgat-ruby** [comment]
@n0mad1995 review please

**Comment by talgat-ruby** [comment]
@Knm24 just resolve conversations and you can merge

**Review by Annurkenova — COMMENTED** [review]
I agree

**Pull request merged**